test/integration/metrics/metrics_test.go

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) F5, Inc.
+//
+// This source code is licensed under the Apache License, Version 2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+
+package metrics
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/nginx/agent/v3/test/integration/utils"
+
+	dto "github.com/prometheus/client_model/go"
+	"github.com/stretchr/testify/suite"
+)
+
+type MetricsTestSuite struct {
+	suite.Suite
+	ctx            context.Context
+	teardownTest   func(testing.TB)
+	metricFamilies map[string]*dto.MetricFamily
+}
+
+func (s *MetricsTestSuite) SetupSuite() {
+	s.ctx = context.Background()
+	s.teardownTest = utils.SetupMetricsTest(s.T())
+	time.Sleep(30 * time.Second)
+	s.metricFamilies = utils.ScrapeCollectorMetricFamilies(s.T(), s.ctx, utils.MockCollectorStack.Otel)
+}
+
+func (s *MetricsTestSuite) TearDownSuite() {
+	s.teardownTest(s.T())
+}
+
+func (s *MetricsTestSuite) TestNginxOSS_Test1_TestRequestCount() {
```

Comment:
Can you remove the Test1/Test2 from the test titles? The name itself will be the identifier